README.md

```diff
@@ -9,22 +9,18 @@ The data is pulled from two different sources:
 - From [`hg.mozilla.org`](https://hg.mozilla.org) for Firefox data.
 - From a [configurable set of Github repositories](repositories.yaml) that use [Glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean).
 
-A web tool to explore the data is available [here](https://telemetry.mozilla.org/probe-dictionary/).
+A web tool to explore the Firefox-related data is available at
+[probes.telemetry.mozilla.org](https://probes.telemetry.mozilla.org/).
+
+CI on this repository handles publishing the data to populate the `probeinfo` API
```

Comment:
I wonder if we should reverse the order of this paragraph with the one above it. Normally I would say that the user facing tools are the most important, but if someone's spelunking through this repo they probably care more about the grotty details. Maybe something like:

```md
Probe Scraper outputs JSON to https://probeinfo.telemetry.mozilla.org. Effectively, this creates a REST API which can be used by downstream tools like [mozilla-schema-generator](https://github.com/mozilla/mozilla-schema-generator) and various data dictionary type applications (see below). An [OpenAPI reference](https://mozilla.github.io/probe-scraper/) to this API is available.

A web tool to explore the Firefox-related data is available at [probes.telemetry.mozilla.org](https://probes.telemetry.mozilla.org/). A project to develop a similar view for Glean-based data
is under development in the [Glean Dictionary](https://github.com/mozilla/glean-dictionary).
```

Thoughts?

Reply:
Looks great. I'll integrate that text.